extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -182,6 +185,13 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /** @override */
+  buildCallback() {
+    const adType = this.element.getAttribute('type');
+    this.config = adConfig[adType];
+    user().assert(this.config, `Type "${adType}" is not supported in amp-ad`);
```

Comment:
We could also consider merging `_a4a_config.js` into `_config.js`.  Not sure if that muddles considerations too much, though.